bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpHandler.java

```diff
@@ -189,6 +189,8 @@ private void cancelPollingJob() {
      * To be called from implementing classes when initializing the device, to start initialization refresh
      */
     protected void initDevice() {
+        updateStatus(ThingStatus.UNKNOWN);
```

Comment:
This could be moved inside the `if` below to avoid two updates in case of configuration error.

Reply:
@jlaur Thanks and done.